rackcity/views/rack_views.py

```diff
@@ -26,3 +30,66 @@ def rack_detail(request, pk):
     if request.method == 'GET':
         serializer = RackSerializer(rack)
         return JsonResponse(serializer.data)
+
+
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+def rack_get(request):
+    """
+    List all racks within specified range.
+    """
+    range_serializer = RackRangeSerializer(data=request.data)
+    if range_serializer.is_valid():
+        try:
+            range_serializer.validate(request.data)
+        except Exception as error:
+            return JsonResponse(
+                {"failure_message": str(error)},
+                status=HTTPStatus.BAD_REQUEST
```

Comment:
Oh I forgot about that. Although NOT_ACCEPTABLE has a nice ring to it, I don't think that's correct because that indicates the request did NOT necessarily fail whereas BAD_REQUEST status indicates bad syntax & that the "client SHOULD NOT repeat the request without modifications"